aksel.nav.no/website/pages/templates/theme-switch/bare-ikon.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  CheckmarkIcon,
+  MonitorIcon,
+  MoonIcon,
+  SunIcon,
+  ThemeIcon,
+} from "@navikt/aksel-icons";
+import { ActionMenu, Button, HStack, Tooltip } from "@navikt/ds-react";
+import { useThemeExample } from "@/web/examples/ThemeExample.context";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+function Example() {
+  const { theme, setTheme } = useThemeExample();
+
+  return (
+    <ActionMenu>
+      <Tooltip content="Bytt fargetema" placement="bottom">
+        <ActionMenu.Trigger>
+          <Button
+            variant="secondary-neutral"
+            data-color="neutral"
+            icon={<ThemeIcon aria-hidden />}
+          />
+        </ActionMenu.Trigger>
+      </Tooltip>
+
+      <ActionMenu.Content>
+        <ActionMenu.Label>Velg fargetema</ActionMenu.Label>
+        <ActionMenu.Group aria-label="Velg fargetema">
+          <ActionMenu.Item
+            icon={<SunIcon />}
+            aria-current={theme === "light"}
+            onSelect={() => setTheme("light")}
+          >
+            <HStack gap="space-24" align="center">
+              <span>Lys</span>
```

Comment:
"Modus" -> "Lys" / "Mørk"
"Tema" -> "Lys**t**" / "Mørk**t**"

Also, looks like we don't need to wrap it in span (?)

```suggestion
              Lyst
```